components/organisms/o-shipping.vue

```diff
@@ -199,10 +196,6 @@ export default {
         required,
         unicodeAlphaNum
       },
-      apartmentNumber: {
-        required,
```

Comment:
can we not remove the required and leave the unicodeAlphaNum validation?

Reply:
Even if we only leave the unicodeAlphaNum validation, we will not be able to click on "continue to payment" if we leave the apartment number field empty.

![image](https://user-images.githubusercontent.com/54149336/117003739-d9988000-acaa-11eb-88b4-9da8304aef53.png)

Reply:
In fact, you can see that PhoneNumber is the only field that is not required in the entire form, that field doesn't have any of the properties that I deleted in apartmentNumber